aten/src/ATen/native/mps/operations/BinaryOps.mm

```diff
@@ -246,7 +248,7 @@ void add_sub_template(const Tensor& self, const Tensor& other, const Scalar& alp
 #define CREATE_MPS_STRUCTURED_BINARY_OP_FUNC(func_out, func_stub, other_type)                   \
 TORCH_IMPL_FUNC(func_out) (const Tensor& self, const other_type& other, const Tensor& output) { \
   TORCH_CHECK(!(self.scalar_type() == ScalarType::Long &&                                       \
-               (std::string(#func_stub) == "power" || std::string(#func_stub) == "atan2")),     \
```

Comment:
I think the check for `self` is still needed (for macOS under 13.2). Otherwise, passing a long input would crash on older OSes. e.g:
```
import torch


device="mps"
a = torch.tensor([1, 2, 3], dtype=torch.int64, device=device)
b = torch.tensor(5, dtype=torch.uint8, device=device)
x = torch.pow(a, b)
print(x)
```